### PR TITLE
DOC-9070: Release Notes for 7.0.1

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -188,12 +188,68 @@ The Couchbase JIRA filter https://issues.couchbase.com/issues/?filter=19680[Couc
...
-* CVE-2021-35943 
+* CVE-2021-35943
 * https://nvd.nist.gov/vuln/detail/CVE-2021-23840[CVE-2021-23840]
 * https://nvd.nist.gov/vuln/detail/CVE-2019-10768[CVE-2019-10768]
 * https://nvd.nist.gov/vuln/detail/CVE-2021-3450[CVE-2021-3450]
-* https://nvd.nist.gov/vuln/detail/CVE-2021-3449[CVE-2021-3449]
\ No newline at end of file
+* https://nvd.nist.gov/vuln/detail/CVE-2021-3449[CVE-2021-3449]
+
+[#release-701]
```
Comment (amarantha-k) [inline modules/release-notes/pages/relnotes.adoc]: We list the latest release at the top of the page, so release 7.0.1 should be before 7.0.0

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -188,12 +188,68 @@ The Couchbase JIRA filter https://issues.couchbase.com/issues/?filter=19680[Couc
...
+== Release 7.0.1 (September 2021)
+
+Couchbase Server 7.0.1 was released in September 2021.
+This maintenance release contains bug fixes.
+
+[#fixed-issues-701]
+=== Fixed Issues
+
+This section highlights the notable issues fixed in this release.
```
Comment (amarantha-k) [inline modules/release-notes/pages/relnotes.adoc]: v7.0 was a major release and hence did not need the fixed issues listed explicitly. For the maintenance releases, we do need to explicitly list the fixed issues. Pls see v6.5.x release notes for reference

**Review by amarantha-k — CHANGES_REQUESTED** [review]
Added comments on the PR

File: modules/release-notes/pages/relnotes.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: I"ve changed the order accordingly. Thanks.

File: modules/release-notes/pages/relnotes.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/release-notes/pages/relnotes.adoc]: I've removed the reference to the filter.

**Review by ritamcouchbase — APPROVED** [review]

**Review by amarantha-k — APPROVED** [review]
looks good from editorial perspective. Waiting on QE review/sign-off.

**Review by arod1987 — COMMENTED** [review]
looks good to me

File: modules/release-notes/pages/relnotes.adoc [hunk 3]
```diff
@@ -5,6 +5,62 @@
...
+
+==== Operator
+
+[#table_fixedissues_v701-operator,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-47678[MB-47678^]
+| *Summary*: Fixed an error occurring during the refresh of the FTS indexer.
```
Comment (mihirkamdarqe) [inline modules/release-notes/pages/relnotes.adoc:26]: This should be reworded to say something like "Fixed an error occurring when running Flex index queries on a setup that uses alternate addresses"

**Pull request merged**